lbrynet/core/Wallet.py

```diff
@@ -20,11 +20,10 @@
 from lbrynet.interfaces import IRequestCreator, IQueryHandlerFactory, IQueryHandler, IWallet
 from lbrynet.core.client.ClientRequest import ClientRequest
 from lbrynet.core.Error import (UnknownNameError, InvalidStreamInfoError, RequestCanceledError,
-                            InsufficientFundsError)
+                                InsufficientFundsError)
```

Comment:
How should i format this to avoid a long line? Is it less ugly to have a `\` at the end of the line? I prefer parentheses to backslashes, but maybe that's just me.